fdb-extensions/src/main/java/com/apple/foundationdb/async/MoreAsyncUtil.java

```diff
@@ -806,6 +807,30 @@ public static boolean isCompletedNormally(@Nonnull CompletableFuture<?> future)
         return future;
     }
 
+    /**
+     * Get a completable future that will either complete within the specified deadline time or complete exceptionally
+     * with {@link DeadlineExceededException}.
+     * @param deadlineTimeMillis The maximum time to wait for the asynchronous operation to complete, specified in milliseconds.
+     * @param supplier The {@link Supplier} of the asynchronous result.
+     * @param <T> The return type for the get operation.
+     * @return A future that will either complete with the result of the asynchronous get operation or
```

Comment:
We have been trying to standardize on some javadoc formatting rules (which we should really create  style-guide for).  All `@param`, `@return`, `@throws`, etc. should be in the form of sentence fragments.  That is, start with a lower case letter and no trailing period, for example:

    @param supplier the {@link Supplier} of the asynchronous result

this is discussed in https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide